packages/chakra-ui/src/TitleField/TitleField.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { FieldProps } from "@rjsf/core";
+
+import { Box, Divider, Heading } from "@chakra-ui/react";
+
+const TitleField = ({ title }: FieldProps) => (
+  <Box mb={1} mt={1}>
+    <Heading as="h5">{title}</Heading>
```

Comment:
we should add a prop override for title sizing if we need it

Reply:
That's a good idea. Where you thinking of any other specific props besides sizing? 

@Tenkir mentioned he wanted to get this into the rjsf project. So @jzander and I are avoiding any deviation from what the other packages are doing until we have the elements displaying properly in the playground. And stripping any specific styling/props. 

We are planning on making it theme-able after we get the elements in the playground. And fixing any specific styling/prop things then. I will take a note of this recommendation and implement it in that phase.